tests/netebpfext_unit/netebpf_ext_helper.cpp

```diff
@@ -61,6 +63,16 @@ _netebpf_ext_helper::_netebpf_ext_helper(
             return;
         }
         platform_initialized = true;
+
+        if (ebpf_random_initiate() != EBPF_SUCCESS) {
+            return;
+        }
+        random_initialized = true;
```

Comment:
is this variable used anywhere apart from being set to true from false?

Reply:
This is used in the below destructor (to identify if we need to invoke the corresponding uninitialize functions).